deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -423,6 +424,26 @@ async def create_new_job(
                 job_type=service,
             )
 
+            # NOTE this happens after the job is already created to have access to the job id, hopefully this doesn't cause any issues with the job starting before the file is uploaded to s3
```

Comment:
I think you should probably do `# Luci (<date>): this happens ...` to keep it consistent with our other repos